src/ui/public/kuery/lib/kuery.peg

```diff
@@ -56,12 +56,32 @@ Term
   }
 
 
-integer "integer"
-  = digits:[0-9]+ { return parseInt(digits.join(''), 10); }
+ /* ----- Core types ----- */
+
+literal "literal"
```

Comment:
For the next language PR it might be worth considering whether we want to allow both quoted and unquoted strings in all places:

* `"response":200` vs `response:200`
* `username:hans` vs `username:"hans"`
* `date:[now to "2017-01-24"]` vs `date:[now to 2017-01-24]`

I have the concern that unquoted strings might lead to ambiguity when the
language grows. On the other hand, always expecting the user to quote strings
might be unfriendly to the non-technical audience. :thinking: